db.js

```diff
@@ -0,0 +1,14 @@
+const { Sequelize } = require('sequelize');
+
+require('dotenv').config();
+
+const client = new Sequelize({
```

Comment:
The `Sequelize` constructor is missing the `username` option. You should include `username: process.env.DB_USERNAME` in the configuration object to ensure the database connection can be established.